### ISSUE TITLE
feat(ui): Make the downloads list scrollable

### ISSUE THREAD

**Issue by LeoniePhiline** [LeoniePhiline/showcase-dl#16]
You can only ever see as many downloads as fit your terminal. This can be an issue on pages with many embeds or on embedded showcases with many individual videos. 

@kiri35 As a workaround, you can zoom out / reduce the terminal font size. This should also fix the "Finishe" label, which, as it sounds like, is cropped in your terminal due to insufficient column width. Smaller font size or larger terminal window should do the trick.

**Comment by giannoxs** [comment]
Hi Leonie, I am trying your tool to download the showcase from Vimeo, I write the command  showcase-dl URL, like https://vimeo.com/showcase/9625437, and I see already the written "Finished" at the top and not any of the downloads. What's wrong? Do I miss something to make it work in the right way?
![Screenshot](https://github.com/LeoniePhiline/showcase-dl/assets/142550790/76089326-561d-4e6b-9a9f-19a65d882cb0)

**Comment by kiri35** [comment]
Thank you!  
Zoom in/out (ctrl+/ctrl-) works for me, if i had less than 30 downlaods :-)

**Comment by giannoxs** [comment]
@kiri35 was it the same problem? see the screenshot attached. https://github.com/LeoniePhiline/showcase-dl/issues/17

**Comment by kiri35** [comment]
> @kiri35 was it the same problem? see the screenshot attached. #17

No. It is for, if there are more downloads activ, than they fit in the terminal window, that is than not high enough.
